distribution/src/bin-filemode-755/hz

```diff
@@ -42,7 +42,7 @@ else
 fi
 
 # HAZELCAST_CONFIG holds the configuration path.
-# If the the path is relative, it is relative to the Hazelcast installation directory (HAZELCAST_HOME).
+# If the path is relative, it is relative to the Hazelcast installation directory (HAZELCAST_HOME).
```

Comment:
This is a newer experimental feature. I'm still working out the quirks for correcting things it flags.

In general, I think that all of these corrections are correct, but it's possible I've made mistakes (I'm human!).

I'm definitely taking feedback and gathering input relating to this and everything else.